lib/src/mt_util.h

```diff
@@ -97,11 +97,30 @@ static inline bool st_is_valid_payload_type(int payload_type) {
 
 void mt_eth_macaddr_dump(enum mtl_port port, char* tag, struct rte_ether_addr* mac_addr);
 
-static inline bool st_rx_seq_drop(uint16_t new_id, uint16_t old_id, uint16_t delta) {
-  if ((new_id <= old_id) && ((old_id - new_id) < delta))
-    return true;
-  else
-    return false;
+static inline bool mt_seq16_greater(uint16_t a, uint16_t b) {
+  return ((int16_t)(a - b)) > 0;
+}
+
+static inline bool mt_seq32_greater(uint32_t a, uint32_t b) {
+  return ((int32_t)(a - b)) > 0;
+}
+
+static inline bool mt_seq64_greater(uint64_t a, uint64_t b) {
+  return ((int64_t)(a - b)) > 0;
+}
```

Comment:
1. Please add comment that this covers wraparounds and it works till difference in values is no bigger then 2^(n-1) where n is the size of variable. after that result is ambiguous. 
2. consider using:
```
static inline bool seq16_greater_u(uint16_t a, uint16_t b) {
    uint16_t diff = (uint16_t)(a - b);
    return (diff & 0x8000u) == 0 && diff != 0;
}
```
instead of cast to signed int. 
The compiler can optimize this whole subtraction and casting and assume a-b > 0 is always true. I think it is exactly what it will do with -O3